setup.py

```diff
@@ -37,7 +37,7 @@
     include_package_data=True,
     install_requires=[
         "dbt-core>=1.1.0",
-        "impyla"
+        "impyla==0.18a5"
```

Comment:
can you just say >=0.18a5 or >=0.18 instead?

Reply:
- We need to say >=0.18a5 . IF we say >=0.18 and package 0.18a1 is say installed, the install_requires condition will satisfy but correct package is still missing.
- Also we need to update readme.md to say pip install correct-impala-version until minor version .18a5 goes public.

Reply:
@TapasSenapati @raghotham updated now.